README.md

```diff
@@ -1 +1,65 @@
-# BuyingCatalogueIdentity
\ No newline at end of file
+# BuyingCatalogueIdentity - Service architecture for the NHS Digital Buying Catalogue Identity .Net Core application
+
+## IMPORTANT NOTES!
+**You can use either the latest version of Visual Studio or .NET CLI for Windows, Mac and Linux**.
+
+### Architecture overview
+This application uses **.NET core** to provide an API capable of runnning Linux or Windows.
+
+### Overview of the application code
+This repository uses **.NET Core** and **Docker**.
+
+It contains three main endpoints
+
+- account/login/returnUrl
```

Comment:
Please review these endpoints.

The login endpoint will be something like 'account/login?returnUrl=<Return URL Here>'